### PR TITLE
Allow to choose the color

### PR DESCRIPTION
**Pull request by janvernieuwe**
Make it so the color can be set in the SimpleCard

### CODE REVIEW

**Comment by janvernieuwe** [comment]
Poke? Can we get this merged?

**Comment by sebbmeyer** [comment]
I should update my emails settings to this request

File: src/Cards/SimpleCard.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function getMessage()
         return [
             "@context" => "http://schema.org/extensions",
             "@type" => "MessageCard",
-            "themeColor" => "0072C6",
+            "themeColor" => isset($this->data['themeColor']) ? $this->data['themeColor'] : "0072C6",
```
Comment (sebbmeyer) [inline src/Cards/SimpleCard.php]: We should check if `themeColor` is not an empty string, but than we can merged it

```suggestion
            "themeColor" => !empty($this->data['themeColor']) ? $this->data['themeColor'] : "0072C6",
```

**Review by sebbmeyer — CHANGES_REQUESTED** [review]
looks good so far but it's need a little adjustment, but then it's a go

**Comment by janvernieuwe** [comment]
Alright, that change looks good to me, thanks for looking at it!

**Review by sebbmeyer — APPROVED** [review]

**Pull request merged**